extension/test/e2e/e2e-test-utils.js

```diff
@@ -24,7 +24,7 @@ async function executeInAdyenIframe (page, selector, executeFn) {
 
 async function initPuppeteerBrowser () {
   return puppeteer.launch({
-    headless: true,
+    headless: process.env.PUPPETEER_BROWSER_HEADLESS || true,
```

Comment:
Any documentation on purpose of this env variable?

Reply:
Removed: 4a79852